packages/gnarly-reducer-erc20/src/models/sequelize.ts

```diff
@@ -0,0 +1,34 @@
+import { makeSequelizeModels as makeGnarlyModels } from '@xlnt/gnarly-core'
+
+const sequelizeModels = (Sequelize: any, sequelize: any) => {
+  const { DataTypes } = Sequelize
+  const { Patch } = makeGnarlyModels(Sequelize, sequelize)
+
+  const ERC20Balances = sequelize.define(
+    'erc20_balances',
+    {
+      id: { type: DataTypes.STRING, primaryKey: true },
+      darAddress: { type: DataTypes.STRING },
```

Comment:
let's replace this with `tokenAddress` or similar; a DAR is a NFT-specific thing (distinct asset registry or whatever it was supposed to mean).